build/build.yml

```diff
@@ -105,6 +105,45 @@ steps:
       BuildDropPath: '$(Build.ArtifactStagingDirectory)'
       PackageName: 'MSSQL for VS Code'
 
+  - task: UseDotNet@2
+    displayName: 'Install .NET Core sdk for signing'
+    inputs:
+      packageType: sdk
+      version: 5.0.x
+      installationPath: $(Agent.ToolsDirectory)/dotnet
+
+  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+    inputs:
+      ConnectedServiceName: 'Code Signing'
+      FolderPath: '$(Build.SourcesDirectory)/.build'
+      Pattern: 'extensions/*.vsix,langpacks/*.vsix'
```

Comment:
Wrong path here, you need to point it at the output path where the VSIX is generated for this repo